aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -194,6 +195,24 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     }
   }
 
+  /**
+   * Validate the Glue table is Iceberg table by checking its parameters. We throw a {@link
+   * NoSuchIcebergTableException} here so that the catalog can decide whether to fail or potentially
+   * use the Spark built-in catalog instead.
+   *
+   * @param table glue table
+   * @param fullName full table name for logging
+   * @throws NoSuchIcebergTableException if the table is not an Iceberg table
+   */
+  static void checkIfTableIsIceberg(Table table, String fullName) {
```

Comment:
I think we should not say anything Spark related in the core library, conceptually it is more like "if the table properties check does not pass, for Iceberg it is equivalent to not having a table in the catalog"